web/src/scenes/team/operations/deleteTeam.generated.ts

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable */
```

Comment:
I suggest to name such folders with graphql code just simply `graphql` to make it look like: `web/src/scenes/team/graphql`. It will be easier to find folder with the gql queries.

Reply:
Renamed. In addition, I propose to group scene components in the `components` directory, because now the components are mixed up with other auxiliary directories, it's not nice, and it's inconvenient to work with.

@paolodamico, @igorosip0v, what do you think?

Reply:
I'm talking about some kind of structure inside the scene directory:
```
/scenes
  /team
    /components
    /graphql
    /services
    /types
```